Header header
int64 GPSTime
float64 enu_x
float64 enu_y
float64 enu_z
float64 Vel_e
float64 Vel_n
float64 Vel_u
float64 ORG_x
float64 ORG_y
float64 ORG_z
